yt_dlp/update.py

```diff
@@ -162,8 +162,12 @@ def _download(self, name, tag):
     def release_name(self):
         """The release filename"""
         label = _FILE_SUFFIXES[detect_variant()]
-        if label and platform.architecture()[0][:2] == '32':
-            label = f'_x86{label}'
+        if label:
+            machine = platform.machine().lower()
+            if machine == 'aarch64' or machine.startswith("arm"):
+                label = f'_{machine}{label}'
+            elif platform.architecture()[0][:2] == '32':
+                label = f'_x86{label}'
```

Comment:
make these distinguishable in _get_variant_and_executable_path

Reply:
If I push a fix for this, would you be able to test it today? Asking so I know whether this can make it into the upcoming release.

Reply:
Sure, I can test today

Reply:
Fixed in a0e827e